src/gui/plugins/component_inspector/Light.qml

```diff
@@ -289,6 +293,37 @@ Rectangle {
         id: grid
         width: parent.width
 
+        RowLayout {
+          Rectangle {
+            color: "transparent"
+            height: 40
+            Layout.preferredWidth: visualizeVisualText.width + indentation*3
+
+            Text {
+              id : visualizeVisualText
+              text: ' Visualize visual enabled'
```

Comment:
I understand what you're trying to convey here, but for someone looking at this casually this may sound weird. How about:

* Visualize (so it matches SDF and msgs)
* Visualization
* View gizmo